numpy/core/src/umath/loops.h.src

```diff
@@ -125,31 +125,20 @@ NPY_NO_EXPORT void
 NPY_NO_EXPORT void
 @S@@TYPE@_fmod(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func));
 
-/**end repeat1**/
-
-NPY_NO_EXPORT void
-U@TYPE@_absolute(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func));
-
-NPY_NO_EXPORT void
-@TYPE@_absolute(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func));
```

Comment:
This commit's diff is just combining `U@TYPE@_absolute` and `@TYPE@_absolute` into `@S@@TYPE@_absolute`, and the same for `_sign`, `_divide`,  and `_remainder`. This is already the pattern we use for other functions in this file